Mathlib/Combinatorics/SimpleGraph/Clique.lean

```diff
@@ -291,6 +428,9 @@ def cliqueSet (n : ℕ) : Set (Finset α) :=
   { s | G.IsNClique n s }
 #align simple_graph.clique_set SimpleGraph.cliqueSet
 
+variable {G H}
```

Comment:
This doesn't seem to be on the port dashboard, but looks reasonable anyway.

Reply:
Yes sorry I added that.